<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CoreSettings</class>
 <widget class="QWidget" name="CoreSettings">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>391</width>
    <height>436</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Settings</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="groupBox_9">
     <property name="title">
      <string>Main account</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_5">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_6">
        <item>
         <widget class="QRadioButton" name="rdbAskAccount">
          <property name="text">
           <string>Ask me which one to use</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="rdbUseAccount">
          <property name="text">
           <string>Always use this account</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="cmbMainAccount">
          <property name="enabled">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="grpFillAccount">
     <property name="title">
      <string>Fill account</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <property name="checked">
      <bool>false</bool>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_8">
      <item>
       <widget class="QComboBox" name="cmbFillAccount"/>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string>Throttle</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_9">
      <item>
       <widget class="QSlider" name="sliderThrottle">
        <property name="minimum">
         <number>1</number>
        </property>
        <property name="maximum">
         <number>160</number>
        </property>
        <property name="singleStep">
         <number>1</number>
        </property>
        <property name="pageStep">
         <number>16</number>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="invertedAppearance">
         <bool>false</bool>
        </property>
        <property name="invertedControls">
         <bool>false</bool>
        </property>
        <property name="tickPosition">
         <enum>QSlider::TicksBelow</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="editThrottle">
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_6">
     <property name="title">
      <string>Software updates</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QCheckBox" name="chkUpdates">
        <property name="text">
         <string>Automatically check for updates at startup</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>rdbAskAccount</tabstop>
  <tabstop>rdbUseAccount</tabstop>
  <tabstop>cmbMainAccount</tabstop>
  <tabstop>grpFillAccount</tabstop>
  <tabstop>cmbFillAccount</tabstop>
  <tabstop>sliderThrottle</tabstop>
  <tabstop>editThrottle</tabstop>
  <tabstop>chkUpdates</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
